index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Ivan Martinez - Portfolio</title>
+   <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+   <header>
+      <nav>
+          <img class="img1" src="resources/image_geometry_1.svg" alt="Mosaic above">
```

Comment:
Kindly use the `<a>` tag for the logo 😃

This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.